src/App.js

```diff
@@ -1,29 +1,39 @@
 import { useState, useEffect } from 'react'
 import Card from "./components/Card"
+
 const App = () => {
   const nav = ["Home", "Exercise1", "Exercise2", "Exercise3", "Exercise4", "Exercise5", "Exercise6", "Exercise7", "Exercise8", "Exercise9"]
   const [userData, setUserData] = useState([]);
```

Comment:
Is this `userData` an array that holds the users fetched from the API? If so, it should be named differently because `userData` sounds like an object holding the user's information. You can choose for example to call it `users` that's enough.

Reply:
userData holds the user's information so it's understandable.